sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -22,6 +22,18 @@
    <version ClassVersion="10" checksum="1379012114"/>
   </class>
 
+  <class name="caf::SRHit" ClassVersion="12">
+   <version ClassVersion="12" checksum="1096003590"/>
+   <version ClassVersion="11" checksum="2966996940"/>
```

Comment:
If I recall correctly, I think you can "flatten" these. So remove the line Version11 line and make the Version12 line say Version11.

Same goes for the other cases below.

Reply:
Or even more, make it Version10 and remove that line too.